tutorial/loadable-components.md

```diff
@@ -7,32 +7,86 @@ nav_order: 16
 
 *This tutorial was contributed by [Athira](https://twitter.com/AthiraMRaju) and [Amogh](https://github.com/ags1773)*
 
-`Loadable components` load all your scripts asynchronously to ensure optimal performances(SSR). Which basically uses a loadable `stats` file created by webpack and from which we get to know which chunks need which JS.
-The `quintype-node-build` has the `default` webpack and babel config. For that to be overridden, we will have to follow the below steps.
+`Loadable components` is a library to solve the React code-splitting client-side, server-side, critical CSS, by loading all the scripts asynchronously to ensure optimal performances.
```

Comment:
could you please add a link  for `Loadable components` `https://loadable-components.com/docs/server-side-rendering/`